contribute/release-policy.md

```diff
@@ -0,0 +1,94 @@
+---
+id: release-policy
+title: Release policy
+---
+
+## Release semantics
+
+The Pulsar project follows a variant of [Semantic Versioning](http://semver.org/spec/v2.0.0.html). Existing releases can expect patches for bugs and security vulnerabilities. New features will target minor releases. The difference is that a major version bump will not carry any special meaning in terms of "big features" included in the release or breaking API changes. Instead, it would simply signal a new long-term support (LTS) release.
```

Comment:
`New features will target minor releases.`  ---- I guess it refers to "major", like 2.10?

Reply:
When we use `marjor.minor.patch` concept, it's the minor.

Reply:
We don't strictly follow the semver spec. We should not introduce incompatible APIs from 3.0.0 to 4.0.0

The LTS version is for stability since we will have many feature and patch releases before bumping a new LTS version. IMO, we should use LTS, feature, and patch releases in the feature. The major release and minor release is confusing.

And the patch release can also happen on a LTS version, like 3.0.1, 3.0.2.

Reply:
Sounds reasonable. Let me try to reword it.